docs/src/views.md

```diff
@@ -244,86 +247,3 @@ large images.
 
 `colorview`'s ability to combine multiple grayscale images is based on
 another view, `StackedView`, which you can also use directly.
-
-## A note on the return types from the views
```

Comment:
I think this section is still useful, might just need to use `channelview` instead.

see also: https://github.com/JuliaImages/juliaimages.github.io/issues/154

Reply:
@johnnychen94 it looks like the examples below were explaining how the lowercase `channelview` returned different types sometimes (which is no longer true, at least for the examples that are provided here). Is that you would like to document here that the input images can have different types but that `channelview` standardizes it?

Reply:
Thanks for explaining; I failed to get the context of this section. Yes, it makes sense to remove it.